tests/js/client/shell/shell-hotbackup-cluster-nonwindows.js

```diff
@@ -56,14 +56,22 @@ function HotBackupSuite () {
         db._createDatabase(dbName);
         db._create(colName);
 
-        const backupGood = arango.POST("/_admin/backup/create", {}).result;
+        const req = arango.POST("/_admin/backup/create", {});
+        if (!require("internal").isEnterprise()) {
+          // Hotbackup is an enterprise feature this call should respond with 404.
+          assertTrue(req.error);
+          assertEqual(req.code, 404);
+          return;
+        }
+        // Entering Enterprise test code
+        const backupGood = req.result;
 
         // Now create the regression. In version 3.6 the entry /arango/Plan/Analyzers did not exist.
 
 
-        assertTrue(readAgencyValueAt("Plan/Analyzers") !== undefined, "We should have analyzers in plan by default.");
+        assertNotUndefined(readAgencyValueAt("Plan/Analyzers"), "We should have analyzers in plan by default.");
```

Comment:
Minor: Anything meaningful we could assert here? Rather then just defined and undefined.

Reply:
In my opinion it is not the job of this test to assert that Analyzers entry is well defined.
The issue was only that the Entry did not exist before, and HotRestore need to create it.